### ISSUE TITLE
Input size

### ISSUE THREAD

**Issue by antoniaaaaaaaa** [jeya-maria-jose/Medical-Transformer#58]
Is the Input size must be 128*128

**Comment by jeya-maria-jose** [comment]
It can be changed in the code.

**Comment by weixj1998** [comment]
@jeya-maria-jose If I plan to use 256*256 size images, do I need to modify the modules in the network? Which parts should be changed? Thanks!

**Comment by zsddd** [comment]
> 可以在代码中更改它。
Hello, I also want to know how to change its size in the code. Thank you very much
